hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/lib/TestMultipleOutputs.java

```diff
@@ -70,6 +76,20 @@ public void testWithCounters() throws Exception {
     _testMOWithJavaSerialization(true);
   }
 
+  @Test(expected=IOException.class)
+  public void testParallelClose() throws IOException, InterruptedException {
```

Comment:
I suggest naming this `testParallelCloseIOException` to make it clear that we are testing an error case.

Reply:
Makes sense, I will change it in my next commit.

Reply:
Addressed